modules/ROOT/partials/configuration/branding.adoc

```diff
@@ -3,7 +3,9 @@
 
 Use the `+branding+` option to disable the "*Powered by Tiny*" link displayed in the status bar for link:{legalpages}/attribution-requirements/[product attribution].
 
-IMPORTANT: Product attribution is required for free and open source users. For information on {productname} attribution requirements, see: link:{legalpages}/attribution-requirements/[Logo & attribution requirements].
+IMPORTANT: Product attribution is required for free users utilizing the Tiny Cloud infrastructure.
+For those using TinyMCE under the MIT license your support for an open source project would be appreciated by keeping attribution turned on in your editor.
```

Comment:
```suggestion
If using TinyMCE under the MIT license, then support for the open source project would be appreciated by keeping attribution turned on in your editor.
```
Pronouns and such shouldn't be used in technical documentation. I'm also not exactly sure what you were trying to say here but hopefully this is the same thing.